prepareComments.js

```diff
@@ -0,0 +1,61 @@
+
+fs = require('fs');
+const data = fs.readFileSync('eslint.json', 'utf8');
+const myArgs = process.argv.slice(2);
+commitId = myArgs[0]
+
+function main(){
+    prepareComment(data);
+}
+
+function createComments(comPath, comCommitId, comLine, comBody, fileName) {
+    console.log("IN FUNCTION -->>"  + comLine );
+    let com = {
+        path: comPath,
+        commit_id: comCommitId,
+        line: comLine,
+        side: "RIGHT",
+        body: comBody
+    };
+    fs.writeFile('eslintComments/file' + fileName + 'jslint.json', JSON.stringify(com), (err) => {
```

Comment:
Expected indentation of 2 spaces but found 4.
This line has a length of 98. Maximum allowed is 80.